cylc/flow/hostuserutil.py

```diff
@@ -113,7 +113,7 @@ def _get_host_info(self, target=None):
         """Return the extended info of the current host."""
         if target not in self._host_exs:
             if target is None:
-                target = socket.getfqdn()
+                target = socket.gethostname()
```

Comment:
whoops again...
Should we think about supporting mac use with cylc...
Could we do something like this
```suggestion
                import platform
                if platform.system() == 'Darwin':
                    target = socket.gethostname()
                else:
                    target = socket.getfqdn()
                   
```

Reply:
> Could we do something like this

No.

Reply:
Fixed, been a while since I last did it!